downstream/modules/platform/ref-ldap-config-on-pah.adoc

```diff
@@ -0,0 +1,177 @@
+:_content-type: REFERENCE
+
+[id="ref-ldap-config-on-pah_{context}"]
+= LDAP configuration on {PrivateHubName}
+
+You must set the following six variables in your {PlatformName} installer inventory file to configure your {PrivateHubName} for LDAP authentication:
+
+* `automationhub_authentication_backend`
+* `automationhub_ldap_server_uri`
```

Comment:
Is "uri" right here, or should it be "url"? Ignore if correct.

Reply:
Yes, that's how it should be.